src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
+    date={article.date}
+    text={article.text}
+  />
```

Comment:
```suggestion
  <Article {...article} />
```
You can pass the whole `article` object to the `props` object using destructuring